README.md

```diff
@@ -1,15 +1,105 @@
 # Validations with `form_for`
 
-## Objectives
+Now that we know Rails automatically performs validations defined on models,
+let's use this information to easily display validation errors to the user.
+
```

Comment:
I think we're assuming this lesson comes after the readme/lab on form_tags? If so, I'd mentioned a little about what we did with form_tags before, and segway into how form_for is a step up in abstraction.